app/presenters/iiif_print/iiif_manifest_presenter_behavior.rb

```diff
@@ -4,26 +4,25 @@ module IiifManifestPresenterBehavior
     extend ActiveSupport::Concern
 
     def search_service
-      Rails.application.routes.url_helpers.solr_document_iiif_search_url(id,
-                                                                         host: request.base_url)
+      Rails.application.routes.url_helpers.solr_document_iiif_search_url(id, host: hostname)
     end
 
     # based on Hyrax::WorkShowPresenter#manifest_metadata
     # expects that individual presenters define #iiif_metadata_fields
-    def manifest_metadata
-      fields = iiif_metadata_fields || []
-      metadata = []
-      fields.each do |field|
-        label = Hyrax::Renderers::AttributeRenderer.new(field, nil).label
-        value = send(field)
-        next if value.blank?
-        value = Array.wrap(value) if value.is_a?(String)
-        metadata << {
-          'label' => label,
-          'value' => Array.wrap(value.map { |f| Loofah.fragment(f.to_s).scrub!(:whitewash).to_s })
-        }
-      end
-      metadata
-    end
+    # def manifest_metadata
```

Comment:
Normally I'm all for removing commented out code.  But, in this case "I'll allow it!" because we're in an active work in progress on this gem.

Reply:
agreed.

Reply:
Though I would prefer a TODO message to follow up on it later.